### PR TITLE
Implement gas estimation of LP-related

### PR DESCRIPTION
**Pull request by lucky7323**
# Description

- Implement gas estimation of LP-related

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/pages/pool/components/add-liquidity-popup.tsx [hunk 1]
```diff
@@ -371,7 +398,95 @@ export const AddLiquidityPopup = ({
...
+        if (step === 1 && token3ApproveEnabled) {
+          // allow token 3
+          estimateApproveFeeAsync(3);
+        }
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [step, isXrp, token1ApproveEnabled, token2ApproveEnabled, token3ApproveEnabled]);
+
+  const estimatedFee = !isXrp
```
Comment (Imchaemin) [inline src/pages/pool/components/add-liquidity-popup.tsx:473]: ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 엄청난 조건문

File: src/pages/pool/components/withdraw-liquidity-popup.tsx [hunk 2]
```diff
@@ -138,45 +149,31 @@ export const WithdrawLiquidityPopup = ({
     symbol: tokensOut?.[1]?.symbol || '',
     spender: vault || '',
     currency: tokensOut?.[1]?.currency || '',
-    enabled: token2Amount > 0 && isXrp,
-  });
-
-  const {
```
Comment (Imchaemin) [inline src/pages/pool/components/withdraw-liquidity-popup.tsx:144]: evm에서 lp토큰은 어프룹 안해도 되던가요..?

**Review by Imchaemin — APPROVED** [review]
테스트 마무리했다면 배포하시죵

File: src/pages/pool/components/withdraw-liquidity-popup.tsx (hunk 2 above)
Comment (lucky7323) [reply src/pages/pool/components/withdraw-liquidity-popup.tsx:144]: 넵

**Pull request merged**